francine/studlist.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <string.h>
+#define SIZE 30
+
+typedef struct{
+    char LName[16];
+    char FName[24];
+    char MI;
+} Nametype;
+
+typedef struct{
+    Nametype name;
+    int ID;
+    char course[8];
+    int year;
+} Studtype;
```

Comment:
@frankie1015 Maybe this should be renamed to `StudType` so it's consistent.

The case where names have all capital letters between words is called **P**ascal**C**ase. This is a
little different from **c**amel**C**ase and snake_case and kebab-case. The other cases are
used in different programming languages as standards

Structure names using `typedef` are usually in **P**ascal**C**ase.